.changeset/pretty-mirrors-peel.md

```diff
@@ -0,0 +1,7 @@
+---
+"@logto/connector-github": minor
+---
+
+Add email address fallback logic:
```

Comment:
first line should be used as a preview line with lowercase starting letter and without ending punctuation mark

Reply:
Updated.